keycloak/sms-provider/src/main/java/org/sunbird/sms/msg91/ProviderDetails.java

```diff
@@ -0,0 +1,18 @@
+package org.sunbird.sms.msg91;
+
+import java.io.Serializable;
+import java.util.List;
+
+public class ProviderDetails implements Serializable {
+    private String sender;
+    private String route;
+    private String country;
+    private List<Sms> sms;
+
+    public ProviderDetails(String sender, String route, String country, List<Sms> sms) {
+        this.sender = sender;
```

Comment:
Can't these move to the config ?

Reply:
@coolbung Yeah, these actually come from configurations, but here I'm just creating a object and which latter I'll convert it to JSON String and send this in POST.